Translations/fr.po

```diff
@@ -10740,10 +10740,10 @@ msgid ""
 "fulfill his endless sacrifices to the Dark ones who scream for one thing - "
 "blood."
 msgstr ""
-"Les armureries de l'Enfer abritent le Seigneur de Guerre du Sang. Dans son "
-"sillage gisaient les corps mutilés de milliers de personnes. Les anges et "
-"les hommes ont été abattus pour accomplir ses sacrifices sans fin envers les "
-"Ténèbres qui crient pour une chose : du sang."
+"Les armureries de l'Enfer abritent le Seigneur de Guerre de Sang. Dans son "
+"sillage gisaient les corps mutilés de multitudes. Anges et hommes ont été "
```

Comment:
```suggestion
"sillage gisaient les corps mutilés de multitudes de personnes. Anges et hommes ont été "
```

Reply:
"Multitudes" alone is voluntary. It’s a way of saying much more literary. It’s the kind of thing that we may hear in the Bible or other legendary stories.